frontend/public/components/graphs/prometheus-graph.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-undef, no-unused-vars */
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import { connectToURLs, MonitoringRoutes } from '../../monitoring';
+
+const getPrometheusUrl = (urls: string[], query: PrometheusQuery[] | string): string => {
+  const base = urls && urls[MonitoringRoutes.Prometheus];
+  if (!base || !query) {
+    return null;
+  }
+
+  const queries = _.isArray(query) ? query : [{query}];
```

Comment:
`_.castArray` does exactly this

```suggestion
  const queries = _.castArray(query);
```

Reply:
I don't think this will work as expected. The `_.castArray` will result in `[query]` instead of `[{query}]`

Reply:
Ah you're right